xgcm/grid.py

```diff
@@ -1399,7 +1401,12 @@ def interp_like(self, array, like):
             except KeyError:
```

Comment:
Why would we continue here if the `_get_axis_coord` raises an error?

Reply:
There could be an axis present in `grid` that is not available for either `array` or `like`. But usually for the axis you are interested in, you would make sure that axis is available for all three.

Reply:
Thanks for the explanation, that makes sense. Could you add this to the comment?